exercises/isogram/example.cpp

```diff
@@ -0,0 +1,30 @@
+#include <ctype.h>
+#include <set>
+#include <utility>
+#include "isogram.h"
+
+using std::string;
+
+namespace isogram {
+ 
+bool is_isogram(string const input_str)
+{
+    std::set<char> chars;
+    std::pair<std::set<char>::iterator, bool> ret;
+    
+    for (char const ch : input_str)
+    {
+        if (isalnum(ch))
+        {
+            ret = chars.insert(tolower(ch));
```

Comment:
It's probably clearer here to use `auto` and remove the type definition for ret above.
```suggestion
            auto ret = chars.insert(tolower(ch));
```